src/v2/Components/NavBar/NavBarPrimaryLogo.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import { ArtsyMarkIcon, Link, LinkProps, color, space } from "@artsy/palette"
+import styled from "styled-components"
+
+const HitArea = styled(Link)`
```

Comment:
Can we use RouterLink here? It will use an `a` tag if the route isn't found within the global routes file. The more we do this up front the less we'll have to do later.